x-pack/plugins/apm/public/components/alerting/transaction_error_rate_alert_trigger/index.tsx

```diff
@@ -97,13 +127,22 @@ export function TransactionErrorRateAlertTrigger(props: Props) {
     />,
   ];
 
+  const chartPreview = (
+    <ChartPreview
+      data={data}
+      yTickFormat={(d: any) => asPercent(d, 1)}
```

Comment:
Can we narrow the type?

```suggestion
      yTickFormat={(d: number) => asPercent(d, 1)}
```

Reply:
Agreed. It's taking a y value so probably `number | null` is better.